cpg_infra/abstraction/base.py

```diff
@@ -333,8 +345,13 @@ def create_machine_account(
         print(f'Creating SA: {name}')
         return name + '@generated.service-account'
 
-    def add_member_to_machine_account_access(
-        self, resource_key: str, machine_account, member, project: str = None
+    def add_member_to_machine_account_role(
+        self,
+        resource_key: str,
+        machine_account,
+        member,
+        role: MachineAccountRole,
+        project: str = None,
     ) -> Any:
         print(f'Allow {member} to access {machine_account}')
```

Comment:
Did you mean to just have a simple `pass` here?

Reply:
This is a DEV CloudInfraBase, so it's supposed to log out what it's doing.